php/Blocks/Loader.php

```diff
@@ -158,6 +166,22 @@ public function editor_assets() {
 			]
 		);
 
+		if ( Settings::ANALYTICS_OPTED_IN_VALUE === get_option( Settings::ANALYTICS_OPTION_NAME ) ) {
```

Comment:
Do you want to factor in any historical opt-in events like we just merged into GB?

Reply:
Hm, good question. 

That might be something for product, but I think for this scope, this might be enough.